docs/about.md

```diff
@@ -61,7 +61,9 @@ What's more, when the creator is free to use their judgment, many new kinds of p
 
 ### **What kind of betting system do you use?**
 
-Manifold Markets uses a special type of automated market marker based on a dynamic pari-mutuel (DPM) betting system.
+Manifold markets is currently using more than one kind of automated market maker.
```

Comment:
I'd directly emphasize "Manifold Markets is currently using two kinds of automated market makers".

- Directly specify how many kinds
- Note the capitalization of Markets
- Bonus: Could link to a writeup of AMMs eg https://www.coindesk.com/learn/2021/08/20/what-is-an-automated-market-maker/

Reply:
One more suggestion: It'd be good to add further structure in the form of numbers here.

Manifold Markets is currently using two kinds of automated market makers.

1. **Dynamic Parimutuel (DPM)**
    Free response markets, ...
 
2. **Constant Function Market Maker (CFMM)**
    ...

Reply:
2. Maniswap